CMakeLists.txt

```diff
@@ -102,10 +102,11 @@ ExternalProject_Add(
     GIT_REPOSITORY https://github.com/NervanaSystems/ngraph
     GIT_TAG v0.6.0
     CMAKE_ARGS 
+        -DNGRAPH_DISTRIBUTED_ENABLE=1 
```

Comment:
Can we make this optional?

Reply:
Made `-DNGRAPH_DISTRIBUTED_ENABLE=1` optional in CMakeLists.txt file